src/components/inputs/InputPasswordToggle/InputPasswordToggle.tsx

```diff
@@ -1,17 +1,20 @@
 import * as React from 'react';
 import classnames from 'classnames';
 import EyeSVG from '../../../svg/eye';
-import IReactComponentProps from '../../../common/structures/IReactComponentProps';
+import ILocalContainerProps from '../../../common/structures/ILocalContainerProps';
 
-interface IState {
+interface IProps extends ILocalContainerProps {
+	onChange?: any;
+	value?: string;
```

Comment:
These two native props were yielding errors because they weren't explicitly provided prior to this PR.